### ISSUE TITLE
Issues (installed via Hoobs)

### ISSUE THREAD

**Issue by nullbreached** [drheck/homebridge-intex#1]
**Describe Your Problem:**
Device is added to Home but control or live data isn’t available. Logs show steam of errors. 

**Logs:**

```

5/3/2023, 11:58:39 AMHoobs PorterSpaposCommand-Xerror: AxiosError: Request failed with status code 400
5/3/2023, 11:58:39 AMHoobs PorterSpa{"errors":{"deviceId":["The value 'undefined' is not valid."]},"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"One or more validation errors occurred.","status":400,"traceId":"00-c77ebbfd458d9f4c9dfc998481fda116-27e7283af294e348-00"}
5/3/2023, 11:58:41 AMHoobs PorterSpapostcommand-URL: https://intexiotappservice.azurewebsites.net/api/v1/command/undefined, data= { sid: 1683129521626, type: 1, data: 8888060F010400D4

```

**Plugin Config:**

{
    "platform": "homebridge-intex",
    "name": "Spa",
    "model": "Intex",
    "refreshInterval": 1,
    "username": " ",
    "password": " ",
    "Filter": true,
    "Bubbles": true
}

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**: 1.04
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by drheck** [comment]
Hello nullbreached,
did you have installed the intex app on your smartphone? You must register an account for your pool and then use the username and password in the plugin.

**Comment by nullbreached** [comment]
Yes. I’ve been using the app for a couple years now and I used the correct login information—same as the app. (iOS)

**Comment by drheck** [comment]
Okay. I have my pool last saturday filled and the plugin has connected and all is fine. I am using homebridge on raspberry.
I have no idea. Sorry.
